src/main/java/seedu/address/model/RepositoryModelManager.java

```diff
@@ -95,6 +98,9 @@ public void showAllItems() {
         updateFilteredItemList(x -> true);
     }
 
+    public ArrayList<Task> returnList() {
+        return new ArrayList<>();
+    }
```

Comment:
The name for this function is misleading as what it does is to return an empty list.
Saw the usage is for the test cases, thus you may consider to use Repository.class that implement the ReadOnlyRepository interface.

Reply:
thank you for the suggestions, I have incorporated it into my code and removed this method accordingly